README.md

```diff
@@ -588,6 +588,19 @@ end
 
 Note though that if a dynamic with the same name exists in your `templates/dynamics/` directory it will get loaded since it has higher precedence.
 
+Templates can be also loaded from sparkle packs by defining the compiler option `sparkle_pack_template`. No extension should be added in this case:
+
+```yaml
+stacks:
+  us-east-1
+    my-stack:
+      template: template_name
+      compiler_options:
+        sparkle_packs:
+          - some-sparkle-pack
+        sparkle_pack_template: true
```

Comment:
I wonder if it'd be better to omit `template` and specify `sparkle_pack_template: template_name` directly for this scenario 🤔

Reply:
Yup, that's the plan we talked about with Patrick today. Pending change tomorrow.